pkg/eventbus/mqtt/client.go

```diff
@@ -124,7 +124,7 @@ func (mq *MQTTClient) InitSubClient() {
 	}
 	subID := fmt.Sprintf("hub-client-sub-%s", timeStr[0:right])
 	subOpts := util.HubclientInit(mq.MQTTUrl, subID, "", "")
-	subOpts.OnConnect = onSubConnect
+	//subOpts.OnConnect = onSubConnect
```

Comment:
onSubConnect will subscribe to all the subTopics on external mqtt broker. Why are we commenting this ?

Reply:
i will sub the default topics in method func (ed *eventbus) subscribe(topic string)

Reply:
onSubConnect connects to all these topics as soon as the client changes state to connected
```SubTopics = []string{
		"$hw/events/upload/#",
		"$hw/events/device/+/state/update",
		"$hw/events/device/+/twin/+",
		"$hw/events/node/+/membership/get",
		"SYS/dis/upload_records",
	}
```
this function func (ed *eventbus) subscribe(topic string) doesn't take care of this right ?

Reply:
yes u are right, i will fix it.